src/node_worker.cc

```diff
@@ -976,5 +983,6 @@ void RegisterExternalReferences(ExternalReferenceRegistry* registry) {
 }  // namespace node
 
 NODE_BINDING_CONTEXT_AWARE_INTERNAL(worker, node::worker::InitWorker)
+NODE_BINDING_PER_ISOLATE_INIT(worker, node::worker::InitWorker)
```

Comment:
Can we not overload the initializers here? I previously thought that `Initialize()` and friends are already quite difficult to understand (what are they initializing?) It would be a good chance to rename them to `CreatePerContextProperties()`/`CreatePerIsolateProperties()` or something like that.